lib/common/options_display.c

```diff
@@ -370,6 +373,12 @@ add_option_metadata_xml(pcmk__output_t *out,
             if (deprecated) {
                 pcmk__add_word(&desc_short_legacy, init_sz,
                                "*** Deprecated ***");
+
+                if (replaced_with != NULL) {
+                    pcmk__g_strcat(desc_short_legacy,
+                                   " *** Replaced with ", replaced_with, " ***",
```

Comment:
I chose not to do `"*** Deprecated (replaced-with X) ***"` because I would not be remotely surprised if some tool is looking for `"*** Deprecated ***"`.

I would have sworn we ran into that with pcs when improving the option listing, but I can't find anything like that in pcs now (even when looking at old versions).